crates/worker/src/partition/state_machine/command_interpreter/mod.rs

```diff
@@ -118,6 +119,8 @@ pub(crate) struct CommandInterpreter<Codec> {
     partition_key_range: RangeInclusive<PartitionKey>,
     latency: Histogram,
 
+    default_invocation_status_source_table: SourceTable,
```

Comment:
Let's add a comment explaining what this field is for.

Reply:
I'll add this in the other PR